.gitignore

```diff
@@ -10,3 +10,6 @@ tags
 tegra/
 *.i
 .download*
+*.caffemodel
+*.prototxt
+build_dir/
```

Comment:
I think it's better not to include it to gitignore

Reply:
It was added for more convenient experiments with trackers in opencv-contrib. I will remove it from gitignore.